apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -94,6 +90,29 @@ onDocumentReady(() => {
     },
     columns: tableColumns(assessmentGroupWork),
   });
+
+  on('submit', 'form.js-popover-form', (event) => {
+    if (!event.currentTarget) return;
+
+    event.preventDefault();
+    $.post(
+      window.location.pathname,
+      $(event.currentTarget).serialize(),
+      function () {
+        refreshTable();
+      },
+      'json',
+    );
+
+    // Immediately close the popover.
+    const popover = event.currentTarget.closest('.popover');
+    if (popover) {
+      // TODO: This won't work in Bootstrap 5, see the following:
+      // https://github.com/twbs/bootstrap/issues/36837
+      $(popover).popover('hide');
+    }
+  });
```

Comment:
If the comment above in behaviors/popover.ts is applied, this could be unnecessary.

Reply:
Maybe add a note to clarify that this is being done without waiting for the result of fetch? Otherwise this will likely be overlooked when this function ends up becoming async'ed.

Reply:
I added a comment in 5b82470, is that what you had in mind?

Reply:
Yes, that looks good. An alternative to disabling the form is to disable the submit button(s), similar to what is done in question submission. That can, of course, be done in a future PR.